pl_bolts/models/rl/ppo_model.py

```diff
@@ -0,0 +1,361 @@
+import argparse
+from typing import List, Tuple
+
+import torch
+import pytorch_lightning as pl
+from torch.optim.optimizer import Optimizer
+from torch.utils.data import DataLoader
+
+from pl_bolts.datamodules import ExperienceSourceDataset
+from pl_bolts.models.rl.common.networks import ActorCategorical, ActorContinous, MLP
+from pl_bolts.utils import _GYM_AVAILABLE
+from pl_bolts.utils.warnings import warn_missing_pkg
+
+if _GYM_AVAILABLE:
+    import gym
+else:
+    warn_missing_pkg('gym')
+
+
+class PPO(pl.LightningModule):
+    """
+    PyTorch Lightning implementation of Proximal Policy Optimization
+    <https://arxiv.org/abs/1707.06347>`_
+    Paper authors: John Schulman, Filip Wolski, Prafulla Dhariwal, Alec Radford, Oleg Klimov
+
+    Model implemented by:
+
+    Example:
+        >>> from pl_bolts.models.rl.ppo_model import PPO
+        ...
+        >>> model = PPO("CartPole-v0")
+
+    Train::
+
+        trainer = Trainer()
+        trainer.fit(model)
+
+    Note:
```

Comment:
Even though we have `Train::` sections in other places in the docs already, I'm sure we don't need this section because this is saying nothing really...
```suggestion
        >>> model = PPO("CartPole-v0")

    Note:
```